wiki/People/The_Team/Beatmap_Nominators/Rules/tr.md

```diff
@@ -0,0 +1,70 @@
+- **Bir şarkının önemli derecede farklı versiyonlarından en fazla 3 tanesi aynı mod içerisinde aynı anda niteliklendirilebilir.** Remiksler, derlemeler, ve karıştırmalar şarkının önemli derecede farklı versiyonlarına dahildir.
+- **Asil üyesi olmadığnız modlardaki beatmapleri, eğer olan bir başkası tarafından aday gösterilmediyse niteliklendirmeyin.** Bu, bir asil aday göstericinin her bir modda sürecin her daim bir parçası olmasını sağlar.
+- **Bir beatmapi mapper ile iletişim kurmadan doğrudan niteliklendirmeyin.** Son aday göstermeden sonra mapperın kendisi bazı değişiklikler yapmış olabilir, dolayısıyla mapper ile irtibat halinde olmadan bir beatmap setini aday göstermek gereksiz diskalifikasyonlara yol açabilir.
+- **Melez beatmap setleri niteliklendirilebilmeleri için her iki modun asil üyeleri tarafından aday gösterilmeli**, aynı zamanda modlama tartışmalarında diğer modların aday göstericilerinin açık onayını kazanmaları gerekir. Bu, sadece asil üyelerin melez setleri aday gösterebileceği anlamına gelir, ve her bir mod için birer asil aday göstericinin (aday göstermek ya da sadece tartışma içerisinde onay vermek olsun fark etmez) sorumluluğu üstleneceğini garantiler. Bu doğrultuda, birden fazla modda uzman olan aday göstericiler birden fazla modda onaylayabilirler.
+
+## Aday gösterme sıfırlama
+
+- **Bir beatmapin aday göstermesi/göstermeleri sıfırlandığında, gönderinin ilk satırı sıfırlamanın kısa bir gerekçesini içermelidir.** Gönderileri linklemek yeterli değildir, alakalı gönderileri özetlemeyi denemelisiniz. Diskalifikasyon durumlarında, sadece bunun talep edildiğini yazmak da yeterli değildir. Gönderilecek çok fazla sorun olmadığı sürece beatmap tartışmalarında gönderilen her bir alakalı sorunu linklemelisiniz.
+- **Eğer aday gösterilen bir beatmap derecelendirilmez sorunlar içeriyorsa, onun aday göstermesi/göstermeleri sıfırlanmalıdır.** Beatmap setini aday göstermeye niyetlenmeseniz de bu geçerlidir.
+- **Beatmapin aday göstermesini/göstermelerini sıfırlamadan önce beatmapin en güncel versiyonuna bakıyor olduğunuzdan emin olun.**
+
+### Diskalifikasyon
+
+- **Sadece asil üyeler beatmapleri diskalifiye edebilir, sebebi ne olursa olsun.** Deneme sürecindeki üyeler diğer herkes gibi bu tarz tartışmalarda yer alabilir ve beatmapleri raporlayabilir.
+- **Diskalifikasyonlar temel olarak derecelendirilemez sorunlar içeren beatmapler için, ya da beatmap setinin sahibinin talebi üzerine uygulanır.**
+- **Beatmap seti sahibinin diskalifikasyon talepleri derhal yerine getirilebilir.** Kendi kendinizi diskalifiye etmek isterseniz, bu sizin için de geçerli.
+- **Çevrimiçi offsetler, etiketler vb. ile hafifletilebilen sorunlar da yine diskalifiye edilmelidir.** Bunun mümkün olmadığı durumlarda, bir NAT üyesiyle irtibata geçin. Çevrimiçi offsetler ve etiketler sadece websitesi için geçerlidir, dolayısıyla Dereceli olmadan önce düzeltilmediğinde oyuncular istemci içerisinde sorun yaşamaya devam edecektir.
```

Comment:
we didn't translate it in several other articles, iirc, since the Turkish locale of the stable version of the game features "offset" most of the time